router/xgress/retransmitter.go

```diff
@@ -140,8 +140,11 @@ func (retransmitter *Retransmitter) retransmitSender() {
 		case retransmit := <-retransmitter.retransmitSend:
 			if !retransmit.isAcked() {
 				if err := retransmitter.forwarder.ForwardPayload(retransmit.x.address, retransmit.payload); err != nil {
-					logger.WithError(err).Errorf("unexpected error while retransmitting payload from %v", retransmit.x.address)
-					retransmissionFailures.Mark(1)
+					// if xgress is closed, don't log the error. We still want to try retransmitting in case we're re-sending end of session
+					if !retransmit.x.closed.Get() {
+						logger.WithError(err).Errorf("unexpected error while retransmitting payload from %v", retransmit.x.address)
+						retransmissionFailures.Mark(1)
+					}
```

Comment:
Might still want to log this at `DEBUG` or `TRACE` when closed?

Reply:
added a trace log